src/app/api/v1/user/update-comm-option/route.ts

```diff
@@ -64,3 +69,9 @@ export async function POST(req: NextRequest) {
     return NextResponse.redirect(AppConstants.SERVER_URL, 301);
   }
 }
+
+function isEmailUpdateCommType(
+  value: string,
+): value is EmailUpdateCommTypeOfOptions {
+  return ["null", "affected", "primary"].includes(value);
```

Comment:
Q: Should we create an `enum` for these since we seem to semi-duplicate some values here and on L42+L45?

Reply:
Also, are there other possible values for `value` that aren't one of these three?

Reply:
TypeScript `enum`s are incompatible with the enum proposal for JS and changes runtime behaviour, and unions of literal strings (which we use here - `EmailUpdateCommTypeOfOptions`) provide the same type safety, so it's recommended to use those.

That type also indicates that indeed, these are the only three valid values :)

That said... See my other comments.